test/functional/feature_bind_extra.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2019 The Bitcoin Core developers
```

Comment:
I am not a lawyer, but my understanding is that:
* The copyright notice here has no legal meaning, and copyright is retained by whoever authored it regardless of what is stated in the file's header.
* "Bitcoin Core developers" is no legal entity, and can't own any copyright. It's just there to indicate "various contributors over time" without the need for being specific.

So there is no problem with using either your own name, or using the collective term - except the latter avoids the need for keeping it up to date as more contributors work on it. Most code I've written for Bitcoin Core is in files with "Bitcoin Core developers" notices, FWIW.

Reply:
> The copyright notice here has no legal meaning, and copyright is retained by whoever authored it regardless of what is stated in the file's header.

The copyright notice is required. Satoshi Nakamoto stated that his code is released under MIT which requires you to keep his name and his orginal copyright notice in place, so when you make changes in the same files created by Satoshi Nakamoto you are required to add your name like this: 

```
Copyright (c) 2008-2010 Satoshi Nakamoto
Copyright (c) 2011-2013 Pieter Wuille
and here comes the MIT statement 
```

The copyright years can't overlap, like you can't say Copyright (c) 2008-2013 Pieter Wuille when there is the clear indication of the years beside the name of Satoshi Nakamoto, this would be a serious legal problem for you. If you don't respect other peoples copyright nobody will respect yours, why would they. 

Saying that copyright notice has no legal meaning is absurd, and under some circumstances if your words turn into action even considered as criminal under international law.

Reply:
This PR is by @vasild. He can legally put any copyright line he wants.

And again, there is no "Bitcoin Core developers" entity. It simply means each author retains his own copyright.

Reply:
@luke-jr if there is no such entity as "Bitcoin Core developers", than this software has very serious copyright issues and each and all contributor by name must be listed in all copyright notices. If there is no _such entity as "Bitcoin Core developers"_ and as @sipa said _copyright notice here has no legal meaning_ than the entire Bitcoin core software is under the **public domain**. 

The MIT license by Satoshi Nakamoto doesn't matter at all because it has "no legal meaning". 

On this end I would strongly suggest to simply just remove all meaningless copyright notices which assigned to an non-existent entity and persons.

Reply:
@JabbaDesilijicTiure My comment was purely about the individual files' copyright notices. The project falls under the MIT license as specified in the COPYING file at the root of the repository.

Reply:
@JabbaDesilijicTiure You don't know what you're talking about. Copyright does not depend on any notice at all. There are plenty of places you can learn more - this isn't the place for it.

Reply:
![luke-jr-the-toothless-bitcoin-core-dev-copyright](https://user-images.githubusercontent.com/71508279/97554459-b83e9a80-19ce-11eb-9100-96116c139074.png)

![lol](https://user-images.githubusercontent.com/71508279/97554606-eae89300-19ce-11eb-8c47-7a9365232a2f.png)

You didn't learned on IRC that I'm not a fan of you? I'm that guy who owns the **https://bitcoin.foundation** domain and I told you that I will make your life very difficult. 

I'm at least **toothsome**! If you don't know what that mean take a mirror front of you or look it up in a dictionary.

Reply:
@JabbaDesilijicTiure I've removed your comment. This repository is obviously not the place for that.